src/Workspaces/Core/Portable/ExternalAccess/Pythia/Api/PythiaSyntaxFactsServiceWrapper.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Pythia.Api
+{
+    internal struct PythiaSyntaxFactsServiceWrapper
```

Comment:
📝 All these new structs can be `readonly`

Reply:
Yup.